test/extended/image_ecosystem/plugin.go

```diff
@@ -378,10 +392,26 @@ func getAdminPassword(oc *exutil.CLI) string {
 	return "password"
 }
 
-func followDCLogs(oc *exutil.CLI, jenkinsNamespace string) {
-	oc.SetNamespace(jenkinsNamespace)
-	oc.Run("logs").Args("-f", "dc/jenkins").Execute()
+// Stands up a simple hello-world pod which can be used for exec commands
+func initExecPod(oc *exutil.CLI) *kapi.Pod {
+	// Create a running pod in which we can execute our commands
+	oc.Run("new-app").Args("https://github.com/openshift/ruby-hello-world").Execute()
```

Comment:
how about just "oc run" with some other image like the jenkins image instead so you don't have to wait for a build?